webapp/src/main/java/com/box/l10n/mojito/service/assetintegritychecker/integritychecker/MessageFormatIntegrityChecker.java

```diff
@@ -23,13 +20,15 @@
  *
  * @author wyau
  */
-public class MessageFormatIntegrityChecker implements TextUnitIntegrityChecker {
+public class MessageFormatIntegrityChecker extends AbstractTextUnitIntegrityChecker {
 
     /**
      * logger
      */
     static Logger logger = LoggerFactory.getLogger(MessageFormatIntegrityChecker.class);
 
+    private static final String REPLACEMENT_CHARS = "!!";
```

Comment:
might want to use some special unicode character to make it safer, like `\x13`

Reply:
Good point!